src/doc/src/reference/unstable.md

```diff
@@ -1215,6 +1215,35 @@ cargo check -Z unstable-options -Z check-cfg=features,names,values
 
 * RFC: [#2906](https://github.com/rust-lang/rfcs/blob/master/text/2906-cargo-workspace-deduplicate.md)
 * Tracking Issue: [#8415](https://github.com/rust-lang/cargo/issues/8415)
+* [Status](https://github.com/rust-lang/cargo/issues/8415#issuecomment-1112618913)
+* [Example Port](https://github.com/clap-rs/clap/pull/3719)
+
+### Testing notes
+
+In preparing to stabilize, we are wanting to better understand
+* If there were any pain points in porting your project
+* Any errors or bugs that you found in testing
+* Performance concerns
+* Gaps in documentation
+* Thoughts on how you feel this feature will work in practice
+
+Please provide feedback on the [tracking issue](https://github.com/rust-lang/cargo/issues/8415)
+or create an issue for any bugs encountered.
+
+You do not have to be running nightly in you project currently or switch to nightly 
+in order help out. You can do a test port [like was done here](https://github.com/clap-rs/clap/pull/3719)
+and that will help us a lot!
```

Comment:
Thinking over this, what I think these testing sections really need is to start with who the target audience is.  For this one:
- Maintainer who has a workspace
- Developing on nightly is optional

Then when talking about nightly, we can point out what our intention is.

Reply:
It seems odd to have a section that is 
```md
Target audience for testing
- Maintainer who has a workspace
- Developing on nightly is optional
```

The two together don't make a ton of sense. I think it could be better as

Target audience for testing
- Maintainer who has a workspace[^1]

[^1]: While your project does not need to be nightly normally to test these changes you do need to use a nightly version of cargo

Reply:
Eh, for me they fit together and burying it in a footnote means people will be more likely to miss it and filter themself out from testing

Reply:
If you feel it is better we will go with it